Modules/_winapi.c

```diff
@@ -1043,9 +1044,14 @@ _winapi_CreateProcess_impl(PyObject *module, Py_UNICODE *application_name,
 
     si.lpAttributeList = attribute_list.attribute_list;
 
+    command_line_copy = PyUnicode_AsWideCharString(command_line, NULL);
```

Comment:
I forgot that this argument can be `None`, so the clinic type has to be `object`. If `PyUnicode_Check(command_line)` is true, then we can copy the string. Otherwise if `command_line != Py_None`, we need to fail with a `TypeError`, e.g. `PyErr_Format(PyExc_TypeError, "CreateProcess() argument 2, command_line, must be str or None, not '%s'", Py_TYPE(command_line)->tp_name)`.